lib/manager/gradle/index.js

```diff
@@ -48,15 +55,8 @@ async function extractPackageFile(content, fileName, config) {
   const gradleFile = path.join(config.localDir, fileName);
   const baseDir = path.dirname(gradleFile);
 
-  if (isProjectRootGradle(fileName)) {
-    await configureUseLatestVersionPlugin(baseDir);
-    const gradleSuccess = await executeGradle(config);
-    if (!gradleSuccess) {
-      return null;
-    }
-  }
-
   const deps = await extractDependenciesFromUpdatesReport(baseDir);
+  gradle.collectVersionVariables(deps, content);
```

Comment:
Do a first pass to collect all the variables used dependencies. And then we look for them in every file